frontend/src/components/App/Notifications/index.tsx

```diff
@@ -132,7 +132,12 @@ function NotificationsList(props: {
             </Grid>
           )}
           <Grid item md={12}>
-            <DateLabel date={notification.date} />
+            <Box display={'flex'} alignItems="center">
+              <Box border={1} p={0.5} mr={1}>
+                {notification.cluster}
+              </Box>
```

Comment:
The cluster name should only be shown if we have more than one cluster available.

Reply:
fixed